.circleci/config.yml

```diff
@@ -418,7 +418,7 @@ workflows:
             - check
       - e2e:
           # Avoids running expensive workflow on PRs
-          <<: *work_branch_workflow_filter
+#          <<: *work_branch_workflow_filter
```

Comment:
you can use `ci/run-full-matrix` to run these, right? or does it not work for it?

Anyway, make sure to remove this once the PR is green

Reply:
I don't think it works for release-1.6